tests/ui/target-feature/wasm-disable-simd.rs

```diff
@@ -0,0 +1,12 @@
+//@ only-wasm32-wasip1
+//@ compile-flags: -Ctarget-feature=-simd128 --crate-type=lib
```

Comment:
Question: does this need `//@ needs-llvm-component: webassembly`?

Reply:
I don't think it's needed. This is my copy from https://github.com/rust-lang/rust/blob/80b74d397fffc9deaafd2b44552ba3a9107b4d5b/tests/ui/target-feature/wasm-relaxed-simd.rs .